app/main/views/outcome.py

```diff
@@ -47,39 +46,32 @@ def award_or_cancel_brief(framework_slug, lot_slug, brief_id):
         )
     }])
 
-    if brief['status'] in ["awarded", "cancelled", "unsuccessful"]:
-        already_awarded = True
-    else:
-        already_awarded = False
-
-        if request.method == "POST":
-            form = AwardOrCancelBriefForm(brief, request.form)
-            if not form.validate_on_submit():
-                errors = {
-                    key: {'question': form[key].label.text, 'input_name': key, 'message': form[key].errors[0]}
-                    for key, value in form.errors.items()
-                }
-            else:
-                answer = form.data.get('award_or_cancel_decision')
-                if answer == 'back':
-                    flash(BRIEF_UPDATED_MESSAGE.format(brief=brief))
-                    return redirect(url_for('.buyer_dos_requirements'))
-                elif answer == 'yes':
-                    return redirect(
-                        url_for('.award_brief', framework_slug=framework_slug, lot_slug=lot_slug, brief_id=brief_id)
-                    )
-                elif answer == 'no':
-                    return redirect(url_for(
-                        '.cancel_award_brief', framework_slug=framework_slug, lot_slug=lot_slug, brief_id=brief_id)
-                    )
-                else:
-                    # We should never get here as the form validates the answers against the available choices.
-                    abort(500, "Unexpected answer to award or cancel brief")
+    form = AwardOrCancelBriefForm(brief)
```

Comment:
Hmm. I think we still need the `if request.method == "POST": ... request.form` data in here somewhere (sorry, I think I missed that on my previous comment).

Reply:
I don't think we do?

`form.validate_on_submit` already checks that it's a POST method, and it also reads directly from `request.form` where appropriate to fill in the fields.

Reply:
That makes sense - I was wondering why the tests still passed 😹 If it means more code gets deleted then I'm all for it!